### PR TITLE
reduce circleci PR build

### PR DESCRIPTION
**Pull request by houqp**

### CODE REVIEW

**Comment by Frenzie** [comment]
Discussion here: https://github.com/koreader/koreader-base/pull/887#issuecomment-484354761

File: .ci/lint_install.sh [hunk 1]
```diff
@@ -8,9 +8,13 @@ source "${CI_DIR}/common.sh"
 echo "pwd: $(pwd)"
 ls
 
-# follow deps are already included in ko docker image
-# travis_retry luarocks --local install luacheck
-# travis_retry luarocks --local install lanes # for parallel luacheck
+# follow deps are already included in ko docker image (used in circleci)
```
Comment (Frenzie) [inline .ci/lint_install.sh]: Alternatively, if we want to keep the Travis builds going for now, we could just switch 'em to the Docker image.

**Comment by Frenzie** [comment]
@NiLuJe Btw, I don't know why Travis and GitLab care so much about log length — just give us the last couple thousand lines I'd say — but the nightly build breaks similarly to Travis. I'm not quite sure why it only happens for Android though.

### Travis

```
/usr/bin/install -c -m 644 /home/ko/base/thirdparty/gettext/build/arm-linux-androideabi/gettext-prefix/src/gettext/gettext-tools/examples/hello-perl/po/ast.po /home/ko/base/thirdparty/gettext/build/arm-linux-androideabi/gettext-prefix/src/gettext-build/share/doc/gettext/examples/hello-perl/po/ast.po

/usr/bin/install -c -m 644 /home/ko/base/thirdparty/gettext/build/arm-linux-androideabi/gettext-prefix/src/gettext/gettext-tools/examples/hello-perl/po/bg.po /home/ko/base/thirdparty/gettext/build/arm-linux-androideabi/gettext-prefix/src/gettext-build/shar

The job exceeded the maximum log length, and has been terminated.
```

### GitLab

```
linux-android/gettext-prefix/src/gettext/gettext-tools/examples/hello-c++-gnome/m4/gnome-gnorba-check.m4 /builds/koreader/nightly-builds/koreader/base/thirdparty/gettext/build/i686-linux-android/gettext-prefix/src/gettext-build/share/doc/gettext/examples/hello-c++-gnome/m4/gnome-gnorba-check.m4
/usr/bin/install -c -m 644 /builds/koreader/nightly-builds/koreader/base/thirdparty/gettext/build/i686-linux-android/gettext-prefix/src/gettext/gettext-tools/examples/hello-c++-gnome/m4/gnome-orbit-check.m4 /builds/koreader/nightly-b
Job's log exceeded limit of 4194304 bytes.
```

You could make verbosity depend on the environment. Detect `GITLAB_CI` and be less verbose. (Or we could add our own custom one in the GitLab config.)

https://docs.gitlab.com/ee/ci/variables/predefined_variables.html

Travis similarly uses `TRAVIS=true` in all builds.

https://docs.travis-ci.com/user/environment-variables/

**Comment by Frenzie** [comment]
Btw, I think we might be able to gain an extra 4 concurrent builds for each of us by enabling CircleCI builds on our own repos. For instance, right now https://github.com/koreader/koreader/commit/8ec42d1e6060833b61645f60acdb6fe0b5fd981a is building on KOReader CircleCI while https://github.com/koreader/koreader/pull/4941 is building on my @Frenzie account. When it's done it transfers the results to the KOReader main repo.

Until that happens, CircleCI won't show anything about it on the PR, which seems a bit odd.

**Comment by NiLuJe** [comment]
Might Android be setting VERBOSE=1 or true somewhere?

Because verbosity *should* be hidden behind that env var.

**Comment by houqp** [comment]
Other build targets like kindle are also getting closer to the log size limit. Kindle build is at 3MB right now, while gitlab's litmit is 4MB. It's strange that android's log is so much larger though.

**Comment by houqp** [comment]
android build log size is at 4.7MB right now.

**Comment by NiLuJe** [comment]
That's libiconv and gettext, for gnulib reasons (among other things).

I'll re --silent them tomorrow.

**Pull request merged**